ballerina/tests/sts_service.bal

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2025, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Fixed in [85735ca](https://github.com/ballerina-platform/module-ballerinax-scim/pull/20/commits/85735ca62055b15e7452f87d2ba6449c266aca25)